server_installation/topics/operating-mode/crossdc.adoc

```diff
@@ -491,6 +511,20 @@ endif::[]
 
 .. The configuration of distributed cache `authenticationSessions` and other caches is left unchanged.
 
+.. It is recommended to add the `remoteStoreSecurityEnabled` property with the value `false` (or eventually `true` if you enabled security
```

Comment:
I think current

    with the value `false`

should be replaced with

    with the value of `false`.

@andymunro Or is the current form correct, too? WDYT?

Reply:
I've changed to the:

```
with the value of `false`
```
Thanks for the suggestion!